tests/test_integrations/test_rapid7.py

```
@@ -20,3 +25,22 @@ def test_integration_active_invalid_key(self, sample_integration_dict):
+            "region": "us",
+            "console_url": "https://console.example.com:3780/",
+            "username": "admin",
+            "password": "password",
+        }
+        integration = sample_integration_dict("rapid7", tokens)
+        service = integration_service_factory.get_service(None, integration)
+        creds = service.generate_python_sdk_creds()
+        assert creds.envs["RAPID7_CONSOLE_URL"] == "https://console.example.com:3780/"
+        assert creds.envs["RAPID7_CONSOLE_USERNAME"] == "admin"
+        from autobotAI_integrations.integrations.rapid7 import Rapid7ConsoleV3Client
+        client = Rapid7ConsoleV3Client(console_url="https://console.example.com:3780/", username="admin", password="password")
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to argument: "password"

(S106)